config/stack/manifests/resources/minimalgcp.ui-schema.yaml

```diff
@@ -1,4 +1,4 @@
-version: 0.0.1
+version: 0.1
```

Comment:
This versioning had been discussed in a previous PR where I updated the ui-schema, I believe we wanted to leave this as `0.0.1` to match the other stacks and per semantic versioning spec <https://semver.org/>

Whenever versions should be updated, it will happen during the publishing pipeline when release branches are cut.